packages/manager-components/global.d.ts

```diff
@@ -0,0 +1,2 @@
+// global.d.ts
+declare const React: ReactDOM;
```

Comment:
What is the purpose of doing this ?

Reply:
we encounter a problem with spec files, as we didn't import react, we have to declare it in global variable for typescript

Reply:
cause we dont use the react var imported